### PR TITLE
Works section(mobile)

### PR DESCRIPTION
**Pull request by nstrvs**
This branch contains two new sections in the mobile version: works, and about me.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,29 +10,154 @@
...
+          <div class="works_images">
+            <img src="works/works01.png" alt="laptop"> 
+          </div>
+          <div class="works_text">
+            <h3>Project name goes here</h3>
+            <div class="tags_works">
+              <p>HTML/CSS</p>
+              <p>Ruby on Rails</p>
+              <p>JavaScript</p>
+            </div>
```
Comment (Whoistolu) [inline index.html:46]: - [ ] [REQUIRED] Note that the list of project technologies is a list `ul` and each of them a `li`. Please ensure to put this list in an unordered list `ul` and each of the technology in a `li` tag. This makes your code more correct and helps screen readers and other technologies that depend on correct semantics to work.

File: index.html [hunk 2]
```diff
@@ -10,29 +10,154 @@
...
+          <div class="works_images">
+            <img src="works/works02.png" alt="laptop"> 
+          </div>
+          <div class="works_text">
+            <h3>Project name goes here</h3>
+            <div class="tags_works">
+              <p>HTML/CSS</p>
+              <p>Ruby on Rails</p>
+              <p>JavaScript</p>
+            </div>
```
Comment (Whoistolu) [inline index.html:60]: - [ ] [REQUIRED] Please refer to my previous comment about embedding the list of technologies in a `ul` and `li` tag.

File: index.html [hunk 3]
```diff
@@ -10,29 +10,154 @@
...
+          <div class="works_images">
+            <img src="works/works03.png" alt="laptop"> 
+          </div>
+          <div class="works_text">
+            <h3>Project name goes here</h3>
+            <div class="tags_works">
+              <p>HTML/CSS</p>
+              <p>Ruby on Rails</p>
+              <p>JavaScript</p>
+            </div>
```
Comment (Whoistolu) [inline index.html:74]: - [ ] [REQUIRED] Please refer to my previous comment about embedding the list of technologies in a `ul` and `li` tag.

File: index.html [hunk 4]
```diff
@@ -10,29 +10,154 @@
...
+          <div class="works_images">
+            <img src="works/works04.png" alt="laptop"> 
+          </div>
+          <div class="works_text">
+            <h3>Project name goes here</h3>
+            <div class="tags_works">
+              <p>HTML/CSS</p>
+              <p>Ruby on Rails</p>
+              <p>JavaScript</p>
+            </div>
```
Comment (Whoistolu) [inline index.html:88]: - [ ] [REQUIRED] Please refer to my previous comment about embedding the list of technologies in a `ul` and `li` tag.

File: index.html [hunk 5]
```diff
@@ -10,29 +10,154 @@
...
+          <div class="works_images">
+            <img src="works/works05.png" alt="laptop"> 
+          </div>
+          <div class="works_text">
+            <h3>Project name goes here</h3>
+            <div class="tags_works">
+              <p>HTML/CSS</p>
+              <p>Ruby on Rails</p>
+              <p>JavaScript</p>
+            </div>
```
Comment (Whoistolu) [inline index.html:102]: - [ ] [REQUIRED] Please refer to my previous comment about embedding the list of technologies in a `ul` and `li` tag.

File: index.html [hunk 6]
```diff
@@ -10,29 +10,154 @@
...
+          <div class="works_images">
+            <img src="works/works06.png" alt="laptop"> 
+          </div>
+          <div class="works_text">
+            <h3>Project name goes here</h3>
+            <div class="tags_works">
+              <p>HTML/CSS</p>
+              <p>Ruby on Rails</p>
+              <p>JavaScript</p>
+            </div>
```
Comment (Whoistolu) [inline index.html:116]: - [ ] [REQUIRED] Please refer to my previous comment about embedding the list of technologies in a `ul` and `li` tag.

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @nstrvs ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] [REQUIRED] It will be nice to have a descriptive Pull Request that is also free of spelling errors (e.g use branch instead of brunch). This is in accordance with professional practice. 

### See your PR Description
PR

### Microverse Descriptive PR [Template](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)


- [ ] [REQUIRED] Kindly stick to the provided [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9) as much as possible.
You need to make your project section fit the width of the mobile version. You seem to have some margin on the right side of your design. See below:

### Your design
your-design

### Figma
figma3

_Check the inline comments under the review for sections to improve._

### Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by nstrvs** [comment]
Hey @Whoistolu thanks for the review, I have already changed the description of the branch. 

Regarding the another request, the Figma template is 320px width, I designed my website in that size, so if you review it like that, it'll fit perfectly.

**Review by Lordkaito — APPROVED** [review]
Hi @nstrvs, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Whoistolu** [comment]
> Hey @Whoistolu thanks for the review, I have already changed the description of the branch.
> 
> Regarding the another request, the Figma template is 320px width, I designed my website in that size, so if you review it like that, it'll fit perfectly.

###Great job @nstrvs 
Happy coding!

**Pull request merged**